spyder/plugins/completion/languageserver/tests/test_client.py

```diff
@@ -217,3 +218,30 @@ def test(a, b):
     # Assert the response has what we expect
     definition = response['params']
     assert 'Test docstring' in definition
+
+
+@pytest.mark.slow
+@pytest.mark.third
+def test_didChangeWorkspaceFolders(lsp_client_and_completion, qtbot):
```

Comment:
I think this test help us to test our side of the call, rather than the pyls one

Reply:
Yeah, that's right, it only tests things in our side because the PyLS doesn't return a response after sending `didChangeWorkspaceFolders` (as you told me).

However, I found an error in our side (we were not deleting workspaces after switching projects), so that's why I decided to add it. Do you want me to move it somewhere else to make things clearer?

Reply:
I think renaming the test would help to clarify that the test is over `send_workspace_folders_change`, rather than on the LSP call

Reply:
Ok, will do.